src/Files.App/Helpers/UIFilesystemHelpers.cs

```diff
@@ -207,18 +200,11 @@ public static async Task CopyItem(IShellPage associatedInstance)
                         if (connection != null)
                         {
                             string filePaths = string.Join('|', associatedInstance.SlimContentPage.SelectedItems.Select(x => x.ItemPath));
```

Comment:
Given that the interop between Files.App and Files.FullTrust is no longer needed, we shouldn't serialize paths to a string anymore. Instead we should pass the `string[]` for paths directly.